feature/gnmi/ate_tests/telemetry_interface_packet_counters_test/telemetry_interface_packet_counters_test.go

```diff
@@ -96,49 +97,58 @@ func TestInterfaceCounters(t *testing.T) {
 	ipv4CounterPath := "/interfaces/interface/subinterfaces/subinterface/ipv4/state/counters/"
 	ipv6CounterPath := "/interfaces/interface/subinterfaces/subinterface/ipv6/state/counters/"
 
-	cases := []struct {
+	type counterCase struct {
 		desc    string
 		path    string
-		counter *telemetry.QualifiedUint64
-	}{{
+		counter func(testing.TB) *telemetry.QualifiedUint64
+	}
+	type counterCases []counterCase
+	cases := make(counterCases, 0)
+	cases = append(cases, counterCase{
 		desc:    "InUnicastPkts",
 		path:    intfCounterPath + "in-unicast-pkts",
-		counter: intfCounters.InUnicastPkts().Lookup(t),
-	}, {
+		counter: intfCounters.InUnicastPkts().Lookup,
```

Comment:
what is the reason to replace Lookup(t) with Lookup?

Reply:
Lookup() is a fatal function which is causing the test to fail if the path is not valid. Definition of "counter" is changed to lookup which is in turn calling lookup() function only for the supported test cases (causing the test to pass)
So the lookup() function will be called in the testing loop via counter(t).